db/version_set.cc

```diff
@@ -995,10 +995,15 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
       storage_info_.num_non_empty_levels_, &storage_info_.file_indexer_,
       user_comparator(), internal_comparator());
   FdWithKeyRange* f = fp.GetNextFile();
+  uint32_t tickers_before[Tickers::TICKER_ENUM_MAX];
+  std::copy(std::begin(get_context.tickers_value),
```

Comment:
what is going on here?? `tickers_value` has garbage data because it's uninitialized, you don't need to remember its garbage values and subtract them later - just initialize it :p.